economytrickle/economytrickle.py

```diff
@@ -108,6 +109,43 @@ def __init__(self, bot):
         await self.bot.say(msgs[settings['TOGGLE']].format(ctx.prefix))
         dataIO.save_json("data/economytrickle/settings.json", self.settings)
 
+    @trickleset.command(name="base", pass_context=True)
+    async def tricklebase(self, ctx, state: str, baseamount: float=None):
+        """Set the base payout and state of base payout
+
+        Current states: on, off, alone
+        
+        state on: Current algorithm + baseamount is trickled
+        
+        state off: Current currency algorithm is trickled
+        
+        state alone: Only the set baseamount is trickled
+        """
+        server = ctx.message.server
+        if state.lower() == self.settings[server.id]["BASE_PAYOUT_STATE"]:
+            msg = "This state is already active\n"       
+        elif state.lower() == "on":
+            msg = "The base payout state has been changed to: on\n"
+            self.settings[server.id]["BASE_PAYOUT_STATE"] = "on"
+        elif state.lower() == "off":
+            msg = "The base payout state has been changed to: off\n"
+            self.settings[server.id]["BASE_PAYOUT_STATE"] = "off"
+        elif state.lower() == "alone":
+            msg = "The base payout state has been changed to: alone\n"
+            self.settings[server.id]["BASE_PAYOUT_STATE"] = "alone"
+        else:
+            msg = "The state you have entered doesn't seem to exist buddy.\n"
+
```

Comment:
make sure to let the user know what's happening.
don't just return without letting them know!
the reason why I asked for the return was so that if the user put in garbage input, it wouldn't overwrite the baseamount as well.